Figures/Helper.cs

```diff
@@ -0,0 +1,271 @@
+                    Console.WriteLine("Enter the height of the triangle");
+                    int height = Convert.ToInt32(Console.ReadLine());
+                    if (isCustom == (int)YN.NO)
+                    {
+                        figureList.Add(new Triangle(height));
+                    }
+                    else
+                    {
+                        figureList.Add(new Triangle(height, Convert.ToBoolean(Convert.ToInt32(userChoice[isFillParam])), Convert.ToChar(userChoice[figurePointParam])));
+                    }
+                    break;
+                case 2:
+                    Console.WriteLine("Enter the height of the rhombus");
+                    height = Convert.ToInt32(Console.ReadLine());
+                    if (isCustom == (int)YN.NO)
+                    {
+                        figureList.Add(new Rhombus(height));
+                    }
+                    else
+                    {
+                        figureList.Add(new Rhombus(height, Convert.ToBoolean(Convert.ToInt32(userChoice[isFillParam])),
+                            Convert.ToChar(userChoice[figurePointParam])));
+                    }
+
+                    break;
+                case 3:
+                    Console.WriteLine("Enter the height of the rectangle");
+                    height = Convert.ToInt32(Console.ReadLine());
+                    Console.WriteLine("Enter the width of the rectangle");
+                    int width = Convert.ToInt32(Console.ReadLine());
+                    if (isCustom == (int)YN.NO)
+                    {
+                        figureList.Add(new Rectangle(width, height));
+                    }
+                    else
+                    {
+                        figureList.Add(new Rectangle(width, height, Convert.ToBoolean(Convert.ToInt32(userChoice[isFillParam])),
+                            Convert.ToChar(userChoice[figurePointParam])));
+                    }
+
+                    break;
+                case 4:
+                    Console.WriteLine("Enter the height of the trapezoid");
+                    height = Convert.ToInt32(Console.ReadLine());
+                    Console.WriteLine("Enter the upper side size of the trapezoid");
+                    width = Convert.ToInt32(Console.ReadLine());
+                    if (isCustom == (int)YN.NO)
+                    {
+                        figureList.Add(new Trapezoid(width, height));
+                    }
+                    else
+                    {
+                        figureList.Add(new Trapezoid(width, height, Convert.ToBoolean(Convert.ToInt32(userChoice[isFillParam])), Convert.ToChar(userChoice[figurePointParam])));
+                    }
+
+                    break;
+                default:
+                    Console.WriteLine("Wrong figure type");
+                    break;
+            }
+        }
+
+        static string[] CollectNewFigureData()
+        {
+            const int MAXPARAMSNUM = 4;
```

Comment:
Consts may only be defined either at a consts file or at the top of the class where the DMs are(usually if they are relevant to that file only and there is no possible usage elsewhere ever.